agent/wsclient/client.go

```diff
@@ -205,6 +205,7 @@ func (cs *ClientServerImpl) ConsumeMessages() error {
 		messageType, message, cerr := cs.Conn.ReadMessage()
 		err = cerr
 		if err != nil {
+			log.Error("Error in ConsumeMessages", "messageType", messageType, "err", err.Error(), "message", message)
```

Comment:
It was intentional that if `err == io.EOF` nothing was logged at the `Error` level. It used to log at that level which resulted in noisy 'err' logs that were simply a normal server close.